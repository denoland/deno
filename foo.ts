function foo(a: number) {
  //
}

foo("sadf");
