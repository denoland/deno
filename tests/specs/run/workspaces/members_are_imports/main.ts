import * as foo from "@deno-test/foo";
import * as bar from "@deno-test/bar";

console.log(foo, bar);
