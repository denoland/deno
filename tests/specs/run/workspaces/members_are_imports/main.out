[Module: null prototype] { value: 42 } [Module: null prototype] { value: 42 }
